File: delivery-info-service/src/test/java/com/potatorider/deliveryinfoservice/domain/DeliverySteps.java
package com.potatorider.deliveryinfoservice.domain;

import com.potatorider.domain.Delivery;
import com.potatorider.domain.DeliveryStatus;
import java.time.LocalDateTime;

public class DeliverySteps {

    public static Delivery makeValidDeliveryWithDeliveryStatus(DeliveryStatus deliveryStatus) {
        String id = "id-1234";
        String orderId = "order1";
        String riderId = "rider-1234";
        String agencyId = "agency-1234";
        String shopId = "shop-1234";
        String customerId = "customer-1234";
        String address = "충주시 호반로...";
        String phoneNumber = "01011112222";
        LocalDateTime orderTime = LocalDateTime.now().minusMinutes(1);
        LocalDateTime pickupTime = LocalDateTime.now().plusMinutes(10);
        LocalDateTime finishTime = LocalDateTime.now().plusMinutes(30);

        return new Delivery(
            id,
            orderId,
            riderId,
            agencyId,
            shopId,
            customerId,
            address,
            phoneNumber,
            "",
            deliveryStatus,
            orderTime,
            pickupTime,
            finishTime);
    }
}

File: delivery-info-service/src/test/java/com/potatorider/deliveryinfoservice/domain/DeliverySteps.kt
package com.potatorider.deliveryinfoservice.domain

import com.potatorider.domain.Delivery
import com.potatorider.domain.DeliveryStatus
import java.time.LocalDateTime

fun makeValidDeliveryWithDeliveryStatus(deliveryStatus: DeliveryStatus?): Delivery {
    val id = "id-1234"
    val orderId = "order1"
    val riderId = "rider-1234"
    val agencyId = "agency-1234"
    val shopId = "shop-1234"
    val customerId = "customer-1234"
    val address = "충주시 호반로..."
    val phoneNumber = "01011112222"
    val orderTime = LocalDateTime.now().minusMinutes(1)
    val pickupTime = LocalDateTime.now().plusMinutes(10)
    val finishTime = LocalDateTime.now().plusMinutes(30)

    return Delivery(
        id,
        orderId,
        riderId,
        agencyId,
        shopId,
        customerId,
        address,
        phoneNumber,
        "",
        deliveryStatus,
        orderTime,
        pickupTime,
        finishTime
    )
}